rest/src/main/java/org/jboss/pnc/rest/utils/BpmNotifier.java

```diff
@@ -69,7 +69,11 @@ public void sendBuildExecutionCompleted(String uri, BuildResult buildResult) {
         String errMessage = "";
         try {
             buildResultRest = new BuildResultRest(buildResult);
-            log.debug("Sending build result to BPM " + buildResultRest.toString() + ".");
+            if (log.isTraceEnabled()) {
+                log.trace("Sending build result to BPM " + buildResultRest.toString() + ".");
+            } else {
+                log.debug("Sending build result to BPM " + buildResultRest.toLogString(100) + ".");
```

Comment:
Shouldn't we rather print the tail of the log? And maybe smaller part?
I'd propose to do smth like log.tail(10)